frontend/awx/resources/templates/WorkflowVisualizer/wizard/NodeAddWizard.tsx

```diff
@@ -162,7 +160,13 @@ export function NodeAddWizard() {
     }
     model.nodes?.push(nodeToCreate);
     controller.fromModel(model, true);
+    controller.getNodeById(nodeToCreate.id)?.setState({ modified: true });
+    controller
+      .getEdgeById(`${state.sourceNode?.getId()}-${nodeToCreate.id}`)
+      ?.setState({ modified: true });
```

Comment:
I don't think we need to set `modified: true` to the newly created node or the edge. 

The saveVisualizer hook detects new nodes through the node's name (probably not the most foolproof way to detect new nodes, but we can look into that) and the newly created edge's source node should be set to `modified: true` up on line 148, that should be adequate enough for save to detect which nodes and edges to POST.